pkg/operator/observe_config.go

```diff
@@ -75,18 +75,18 @@ func NewConfigObserver(
 		},
 		listers: Listers{
 			imageConfigLister: configInformer.Config().V1().Images().Lister(),
-			configmapLister:   kubeInformersForOpenshiftCoreOperators.Core().V1().ConfigMaps().Lister(),
+			configmapLister:   kubeInformersForOperator.Core().V1().ConfigMaps().Lister(),
 			buildConfigLister: configInformer.Config().V1().Builds().Lister(),
 		},
 	}
 
 	c.operatorConfigSynced = operatorConfigInformer.Informer().HasSynced
-	c.configmapSynced = kubeInformersForOpenshiftCoreOperators.Core().V1().ConfigMaps().Informer().HasSynced
+	c.configmapSynced = kubeInformersForOperator.Core().V1().ConfigMaps().Informer().HasSynced
 	c.imageConfigSynced = configInformer.Config().V1().Images().Informer().HasSynced
 	c.buildConfigSynced = configInformer.Config().V1().Builds().Informer().HasSynced
 
 	operatorConfigInformer.Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftCoreOperators.Core().V1().Namespaces().Informer().AddEventHandler(c.eventHandler())
+	kubeInformersForOperator.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
```

Comment:
Watch the config maps for changes to `openshift-controller-manager-images`

Reply:
@sanchezl you might also have input here.